### PR TITLE
[TL/LT][88355376] add error handling

### PR DESCRIPTION
**Pull request by Teapane**
[Story](https://www.pivotaltracker.com/story/show/88355376) 

Add error handling to deal with no images.

### CODE REVIEW

File: jquery.lazyload.js [hunk 1]
```diff
@@ -126,6 +126,9 @@
                                 var elements_left = elements.length;
                                 settings.load.call(self, elements_left, settings);
                             }
+                        }).bind("error", function() {
+                            var fallback = $self.attr("data-fallback");
+                            $self.attr("src", fallback);
```
Comment (sbrauer) [inline jquery.lazyload.js:131]: Does anything weird happen if data-fallback isn't present?
Would it be safer to only set src if there is data-fallback?

Comment (luketower) [inline jquery.lazyload.js:131]: Good idea. We could just use the original placeholder (grey box) that is displaying currently, if there is no `data-fallback`.

Comment (c0) [inline jquery.lazyload.js:131]: :+1: It would be easy to implement: `var fallback = $self.attr("data-fallback") || settings.placeholder;`

**Comment by c0** [comment]
:+1:

**Pull request merged**